cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -26,7 +26,7 @@ import (
 	"go.thethings.network/lorawan-stack/pkg/errors"
 	"go.thethings.network/lorawan-stack/pkg/ttnpb"
 )
-
+var errMissingParam = errors.DefineInvalidArgument("missing_params", "missing token or reference")
```

Comment:
This doesn't look quite right:

- Spacing is off
- Not located "as close as possible to where used"